mozilla/fx-private-relay
# MPP-2429 - Add white background to text (quick fix)

Follow up for PR https://github.com/mozilla/fx-private-relay/pull/2956 based on input from Andrei.
 


# New feature description



# Screenshot (if applicable)
 
![image](https://user-images.githubusercontent.com/3924990/210699253-7e617ed6-40a0-4676-a166-74aaed4cf2b8.png)


# How to test

Without a true phone number registered, visit the phones dashboard. You should be able to see the white background on the new warning. Thanks! 

# Checklist (Definition of Done)
